docs/static/breaking-changes.asciidoc

```diff
@@ -1,310 +1,38 @@
-|`/usr/share/logstash`
-|===
-
-A complete directory layout is described in <<dir-layout>>. This will likely impact any scripts that you may have written
-to support installing or manipulating Logstash, such as via Puppet.
-
-[float]
-==== Default Logging Level
-
-The default log severity level changed to `INFO` instead of `WARN` to match Elasticsearch. Existing logs
-(in core and plugins) were too noisy at the `INFO` level, so we audited our log messages and switched some of them to
-`DEBUG` level.
-
-You can use the new `logstash.yml` file to configure the `log.level` setting or continue to pass the new
-`--log.level` command line flag.
-
-[source,bash]
-----------------------------------
-bin/logstash --log.level warn
-----------------------------------
-
-[float]
-==== Plugin Manager Renamed
-
-`bin/plugin` has been renamed to `bin/logstash-plugin`. This occurred in Logstash 2.3 and it was mainly prevent `PATH` being
-polluted when other components of the Elastic Stack are installed on the same machine. Also, this provides a foundation
-for future change which will allow Elastic Stack packs to be installed via this script.
-
-Logstash 5.0 also adds a `remove` option, which is an alias for the now-deprecated `uninstall` option.
-
-As with earlier releases, the updated script allows both online and offline plugin installation. For example, to install a
-plugin named “my-plugin”, it’s as simple as running:
+==== Command Line Interface behavior
 
-[source,bash]
-----------------------------------
-bin/logstash-plugin install my-plugin
-----------------------------------
+* The `-e` and `-f` CLI options are now mutually exclusive. This also applies to the corresponding long form options `config.string` and 
+  `path.config`. This means config provided via `-e` will no longer be appended to the config provided via `-f`.
+* Configuration provided with `-e` or `config.string` will not be appended with `stdin` input and `stdout` output automatically.
 
-Similar to the package changes, this is likely to impact and scripts that have been written to follow Logstash
-installations.
+==== List of plugins bundled with Logstash
 
-Like earlier releases of Logstash, most plugins are bundled directly with Logstash, so no additional action is required
-while upgrading from earlier Logstash releases. However, if you are attempting to install a non-bundled plugin, then make
-sure that it supports Logstash 5.0 before upgrading!
-
-[float]
-==== Logstash with All Plugins Download
-
-The Logstash All Plugins download option has been removed. For users previously using this option as a convenience for
-offline plugin management purposes (air-gapped environments), please see the <<offline-plugins>> documentation page.
-
-There were 17 plugins removed from 5.0 default bundle. These plugins can still be installed manually for use:
+The following plugins were removed from 5.0 default bundle based on usage data collected by us. These plugins can still be installed 
```

Comment:
Suggest changing to say:  "The following plugins were removed from the 5.0 default bundle based on usage data. You can still install these plugins manually:"